app/javascript/app/components/country-ghg-emissions/country-ghg-emissions.js

```diff
@@ -110,6 +123,24 @@ class CountryGhgEmissionsContainer extends PureComponent {
     }
   };
 
+  handleRemoveTag = tagData => {
```

Comment:
I was wondering, do you think this could be done inside the Legend Chart Component so we don't have to repeat the code?

Reply:
Yes it can, I wasn't sure which way I preferred as there were some differences at first. I think you're right, I will move it.